lepton/package.go

```diff
@@ -530,13 +517,17 @@ func CreatePackageFromRun(newPkg string, version string, newconfig *types.Config
 	c.Boot = ""
 	c.UefiBoot = ""
 	c.Kernel = ""
+	c.KlibDir = ""
 	c.NanosVersion = ""
 	c.VolumesDir = ""
-	c.NameServers = []string{}
+	c.NameServers = nil
 	c.LocalFilesParentDirectory = ""
 	c.ProgramPath = ""
 	c.CloudConfig = types.ProviderConfig{}
-	c.RunConfig = types.RunConfig{}
+	c.RunConfig = types.RunConfig{
```

Comment:
Because when I tried `ops pkg from-run -c config.json app` it deleted runConfig that was present on  `config.json` file from the produced manifest file. While I can handle canIpForward a bit better (include it only if true) we need at least the klibs the package uses. In any case I can revert this change (and add the missing pieces back manually)

Reply:
i see, i forgot we had the ability to stick RunConfig into a package but a quick scan through the repo show quite a few examples of this (memory settings for jvm, ports, etc.) so I think adding this support is ok but it should only be adding whatever the user has explicitly specified in their config via json or flags - if you remove the line nil'ing it out (L539) it works but adds in a bunch of defaults; turning off buildImageFlags in cmd/cmd_pkg removes most of them but I still see Accel and Memory showing up - if you rework this to only add in what is actually specified by the user in the config we can merge

Reply:
The pr was cleaned up and the package manifest file is being handled separately. It processes the merged config as usual, but it ends up using only a subset of the config that is general and makes sense for the package manifest by default. Let me know if it makes sense or other changes are required. Atm it handles `from-run` only, will take care of `from-pkg` and `from-docker` later on, if needed, once this is accepted/merged.